.github/workflows/native-cron-build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: mvn -B install -DskipTests -DskipITs -Dno-format
 
       - name: Run integration tests in native
-        run: mvn -B  --settings azure-mvn-settings.xml verify -f integration-tests/pom.xml -DskipTests  -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j
```

Comment:
ooops!

Reply:
:astonished:

Reply:
So I think @geoand supposed that just dropping `-DskipITs` was enough and to be honest I'm a bit surprised myself. But I tested it and `-DskipTests` also disables failsafe.

Reply:
Indeed, my bad 🙃